codalab/worker/main.py

```diff
@@ -45,15 +45,15 @@ def parse_args():
     )
     parser.add_argument(
         '--cpuset',
-        type=str,
+        type=cpuset_args,
```

Comment:
This automates the logic of passing the string form of the arg to the `type` function, and sets the arg value as the return value of that function. (in this case our custom parsing function)